code/game/machinery/ecto_sniffer.dm

```diff
@@ -19,21 +19,26 @@
 	wires = new/datum/wires/ecto_sniffer(src)
 
 /obj/machinery/ecto_sniffer/attack_ghost(mob/user)
-	. = ..()
```

Comment:
What does parent do? Why did you remove it?

Reply:
examine iirc

Reply:
Looked at other ghost actions and saw that they didn't include it, this allows ghosts who have inquisitiveness enabled to still interact with the sniffer. Doesn't stop examine.